driver-core/src/test/java/com/datastax/driver/core/ControlConnectionTest.java

```diff
@@ -69,7 +69,7 @@ public long nextDelayMs() {
         };
 
         try {
-            ccm = CCMBridge.builder("test").withNodes(2).build();
```

Comment:
Most tests that use CCM with more than one node couldn't use `PerClassSingleNodeCluster` before for obvious reasons; they can now be refactored with `CCMTestsSupport`.
Tests that use `CCMBridge.CCMCluster` could also be refactored, `CCMCluster` being merely an encapsulation of a `CCMBridge` instance + a `Cluster` instance.